omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -81,7 +81,7 @@ def elastic_search_major_version
         # Decorate any JSON parsing exception or numeric exceptions when accessing and parsing the version field.
         raise "Unable to parse elasticsearch response #{e}"
       end
-      raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2, 5 and 6." if version != 6 && version != 5 && version != 2
+      raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2, 5 and 6." if version != 7 && version != 6 && version != 5 && version != 2
```

Comment:
```suggestion
      raise "Unsupported elasticsearch version of #{version}. There is currently support for the major versions of 2, 5 and 6." unless [2, 5, 6, 7].include?(version)
```

Reply:
s/There is currently support for the major versions of 2, 5 and 6./There is currently support for the major versions of 2, 5 6 and 7.